app/Http/Controllers/Web/DevicesController.php

```diff
@@ -118,14 +106,6 @@ public function handleControlRequest(Request $request, string $action, int $devi
         return redirect()->route('devices');
     }
 
-    private function currentUser(): User
```

Comment:
This function is still needed here.  It gets called from the `add()` function in this class as well.